ui/admin/app/routes/onboarding/quick-setup/create-resources.js

```diff
@@ -58,46 +58,53 @@ export default class OnboardingQuickSetupCreateResourcesRoute extends Route {
   @notifyError(function () {
     return this.intl.t('errors.quick-setup-failed.description');
   })
-  async createResources(hostAddress, targetPort) {
+  async createResources(targetAddress, targetPort) {
     try {
-      await this.createOnboardingResourcesAndRedirect(hostAddress, targetPort);
+      await this.createOnboardingResourcesAndRedirect(
+        targetAddress,
+        targetPort
+      );
     } catch (e) {
       this.router.replaceWith('scopes.scope', 'global');
       throw new Error();
     }
   }
 
-  async createOnboardingResourcesAndRedirect(hostAddress, targetPort) {
+  async createOnboardingResourcesAndRedirect(targetAddress, targetPort) {
     const { org, project, hostCatalog, hostSet, host, target, role } =
       this.currentModel;
-    // The Procedure
+
+    // Persist org and project
     await org.save();
     project.scopeID = org.id;
     await project.save();
-    hostCatalog.scopeID = project.id;
-    await hostCatalog.save();
-    hostSet.host_catalog_id = hostCatalog.id;
-    await hostSet.save();
-    host.host_catalog_id = hostCatalog.id;
-    if (hostAddress) {
-      host.address = hostAddress;
-      await host.save();
-      await hostSet.addHosts([host.id]);
+
+    if (targetAddress) {
+      // Format target object and persist it
       target.scopeID = project.id;
+      target.address = targetAddress;
       target.default_port = targetPort;
       await target.save();
-      await target.addHostSources([hostSet.id]);
+      // Format role and role grants objects and persist them
       role.scopeID = org.id;
       role.grant_scope_id = project.id;
       await role.save();
       await role.saveGrantStrings([
         `type=target;actions=list`,
-        `id=${target.id};actions=authorize-session`,
+          `id=${target.id};actions=authorize-session`,
```

Comment:
I think formatting got messed up here

Reply:
You are right. I updated the PR with this corrected thanks for the catch